keras_cv/models/stable_diffusion/stable_diffusion.py

```diff
@@ -60,6 +64,7 @@ def __init__(
         self._image_encoder = None
         self._text_encoder = None
         self._diffusion_model = None
+        self._diffusion_model_ptp = None
```

Comment:
can we rename the `ptp` variable name?  Prefer to avoid unnecessary acronyms.  Can we be more descriptive with what the modified forward pass does somehow in the name?  Or even just say `diffusion_model_prompt_to_prompt`?

Reply:
Fixed!